Doxygen/CMakeLists.txt

```diff
@@ -42,8 +40,14 @@ if (NOT DOXYGEN_INTERNAL_DOCS)
     set(DOXYGEN_PREDEFINED "__TASMANIAN_DOXYGEN_SKIP;__TASMANIAN_DOXYGEN_SKIP_INTERNAL;Tasmanian_ENABLE_BLAS;Tasmanian_ENABLE_CUDA;Tasmanian_ENABLE_MPI")
 endif()
 
+### Add the git-hash to the main page, suppress at release (the stable page needs no hash)
+file(READ "${CMAKE_CURRENT_SOURCE_DIR}/../README.md" Tasmanian_readme)
+string(REPLACE "# Tasmanian" "# Tasmanian\n    * *Generated from git commit ${Tasmanian_git_hash}*" Tasmanian_readme ${Tasmanian_readme})
+file(WRITE "${CMAKE_CURRENT_BINARY_DIR}/README.md" "${Tasmanian_readme}")
```

Comment:
Why did you prefer this over `configure_file`?

Reply:
Github automatically renders README.md under the code of the main page. I don't want to have `@Tasmanian_git_hash@` or `${Tasmanian_git_hash}`, I want that readme to be  be clean.

Reply:
Oh right